ui/yarn.lock

```diff
@@ -17783,11 +17597,6 @@ run-queue@^1.0.0, run-queue@^1.0.3:
   dependencies:
     aproba "^1.1.1"
 
-rw@1:
```

Comment:
also installed in initial d3 install - removing all lump d3 installations and now just installing individual packages required